Library/Formula/wordgrinder.rb

```diff
@@ -0,0 +1,15 @@
+class Wordgrinder < Formula
+  homepage "http://wordgrinder.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/wordgrinder/wordgrinder/wordgrinder-0.5.2.tar.bz2"
+  sha1 "4bcc600fbd110d9d87961feab15045b9b518b2db"
+
+  depends_on "lua"
+  depends_on "ncurses" => :build
```

Comment:
On OSX, the makefile looks in the Homebrew directory (with the prefix command) for ncurses, and fails if it's not installed through Homebrew.
If it's really preferred to use the version with OSX, I can email the guy who made WordGrinder and find out if version 5.9 is needed or if he's using stuff that's been added since 5.7.